pkg/test/framework/runtime/components/galley/native.go

```diff
@@ -165,6 +176,7 @@ func (c *nativeComponent) Reset() error {
 	a.DisableResourceReadyCheck = true
 	a.ConfigPath = c.configDir
 	a.MeshConfigFile = c.meshConfigFile
+	a.IntrospectionOptions.Port = 9877
```

Comment:
Is this needed due to collision? Can you put a comment for this?

Reply:
yep, to prevent collision, comments added